packages/fxa-auth-server/test/local/email/delivery.js

```diff
@@ -206,4 +207,50 @@ describe('delivery messages', () => {
         assert.equal(log.info.args[1][1].domain, 'aol.com');
       });
   });
+
+  it('should log account email event (emailDelivered)', async () => {
+    const stub = sinon
+      .stub(emailHelpers, 'logAccountEventFromMessage')
+      .returns(Promise.resolve());
+    const log = mockLog();
+    const mockMsg = mockMessage({
+      notificationType: 'Delivery',
+      delivery: {
+        timestamp: '2016-01-27T14:59:38.237Z',
+        recipients: ['jane@aol.com'],
+        processingTimeMillis: 546,
+        reportingMTA: 'a8-70.smtp-out.amazonses.com',
+        smtpResponse: '250 ok:  Message 64111812 accepted',
+        remoteMtaIp: '127.0.2.0',
+      },
+      mail: {
+        headers: [
+          {
+            name: 'X-Template-Name',
+            value: 'verifyLoginEmail',
+          },
+          {
+            name: 'X-Flow-Id',
+            value: 'someFlowId',
+          },
+          {
+            name: 'X-Flow-Begin-Time',
+            value: '1234',
+          },
+          {
+            name: 'X-Uid',
+            value: 'en',
+          },
+        ],
+      },
+    });
+
+    await mockedDelivery(log).handleDelivery(mockMsg);
+    sinon.assert.calledOnceWithExactly(
+      emailHelpers.logAccountEventFromMessage,
+      mockMsg,
+      'emailDelivered'
+    );
+    stub.restore();
```

Comment:
If the test fails, it seems like the stub will not be restored. In this particular test suite I don't think that would be an issue, but I have on rare occasion seen this sort of thing cause problems. An easy work around is making a sinon sandbox and then calling sandbox.restore afterEach.